lib/diagnostic.js

```diff
@@ -13,12 +13,28 @@
 
 'use strict';
 
-const fs = require('fs'); // jshint ignore: line
+let fs = require('fs'); // jshint ignore: line
 
-const sumLines = (filename, callback) => {
+let fileName = process.argv[2];
+
+const sumLines = fs.readFile(fileName, (err, data) => {
   /* your response here */
-  callback();
-};
+  if(err){
+    console.log(err+': File failed to load. Please check file path.');
+  }
+  let lines = data.toString().split('\n');
+  let sum = 0;
+  for(let line in lines) {
+    if(lines[line]/1 == lines[line])
+    {
+      sum += lines[line]/1;
+    }
+    else {
+      console.log('line '+lines[line]+' is not a number');
+    }
+  }
+  console.log('The sum of the numeric lines equals '+sum);
```

Comment:
You are console.logging the answer instead of doing what the directions say you should do...